ibis/backends/pandas/tests/test_udf.py

```diff
@@ -94,11 +94,6 @@ def zscore(series):
     return (series - series.mean()) / series.std()
 
 
-@udf.elementwise([], dt.int64)
```

Comment:
This was an oversight from the vectorized udf changes. We do not support return a scalar from elementwise udf anymore.